packages/framework/src/Resources/styles/admin/components/table/main.less

```diff
@@ -19,13 +22,35 @@
     }
 
     td, th {
-        padding: 14px 17px;
+        padding: @table-main-padding-vertical @table-main-padding-horizontal;
     }
 
     thead {
         td, th {
-            padding-top: 14px;
-            padding-bottom: 14px;
+            padding-top: @table-main-padding-vertical;
+            padding-bottom: @table-main-padding-vertical;
+        }
+    }
+
+    &__column-with-info {
+        display: block;
+        padding-right: @table-main-padding-horizontal;
+    }
+
+    &--with-inline-form {
+        th {
+            padding-right: 2*@table-main-padding-horizontal;
+        }
+
+        td {
+            position: relative;
+            padding: @table-main-padding-vertical 2*@table-main-padding-horizontal @table-main-padding-vertical 10px;
```

Comment:
Can we consider create new variable for `2*@table-main-padding-horizontal`? Its name could be for example `@table-main-padding-horizontal-larger`.
I suggest it because I see another `2*@table-main-padding-horizontal` at row 42.

Reply:
fixed